### PR TITLE
Fix all git commit hook issues.

### PR DESCRIPTION
**Pull request by KatalinaBobowik**

### CODE REVIEW

File: scripts/HGDP_1KG_HailExploration.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
-from hail.plot import show
-
+from bokeh.palettes import turbo  # pylint: disable=no-name-in-module
 import hail as hl
+
 hl.init()
 
-mt = hl.read_matrix_table("gs://gcp-public-data--gnomad/release/3.1/mt/genomes/gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt")
+mt = hl.read_matrix_table(
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'  # noqa: E501 pylint: disable=line-too-long
```
Comment (lgruen) [inline scripts/HGDP_1KG_HailExploration.py:16]:
```suggestion
    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
```

File: scripts/hail_batch/hgdp_1kg_pca_loadings_generate.py [hunk 2]
```diff
@@ -2,15 +2,14 @@
 
 import click
 import pandas as pd
-from hail.methods import hwe_normalized_pca
 import hail as hl
 
-GNOMAD_HGDP_1KG_MT = 'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+GNOMAD_HGDP_1KG_MT = 'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'  # noqa: E501 pylint: disable=line-too-long
```
Comment (lgruen) [inline scripts/hail_batch/hgdp_1kg_pca_loadings_generate.py:7]:
```suggestion
GNOMAD_HGDP_1KG_MT = (
    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
)
```

File: scripts/hail_batch/main.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
 
 batch = hb.Batch(name='generate PCA loadings', backend=service_backend)
 
-dataproc.hail_dataproc_job(
+dataproc.hail_dataproc_job(  # pylint: disable=unexpected-keyword-arg
```
Comment (lgruen) [inline scripts/hail_batch/main.py:19]: If you remove the quotes from `num_secondary_workers='100'`, you probably don't need this linter override?

```suggestion
dataproc.hail_dataproc_job(
```

**Review by lgruen — CHANGES_REQUESTED** [review]

File: scripts/hail_batch/main.py (hunk 3 above)
Comment (KatalinaBobowik) [reply scripts/hail_batch/main.py:19]: on line 25, there's an option for 'job_name', which is not in the latest analysis-runner release (but it is in the example I took it from).

Comment (lgruen) [reply scripts/hail_batch/main.py:19]: Thanks a lot for pointing that out. This should be fixed after updating to 1.4.0 of the analysis-runner (see https://github.com/populationgenomics/analysis-runner/pull/74).

You should still fix the type of the `num_secondary_workers` parameter to match the function declaration though.

**Review by lgruen — APPROVED** [review]

**Pull request merged**